extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -100,6 +103,17 @@ protected function renderItem( $type, array $data ) {
 		return $result;
 	}
 
+	protected function renderData( $nodeData ) {
+		if (
+			!empty( $nodeData['collapse'] ) &&
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "collapse" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)